docs/source/reference/tpu.rst

```diff
@@ -0,0 +1,183 @@
+.. _tpu:
+
+Cloud TPU
+================================
+
+SkyPilot supports running jobs on `Cloud TPU <https://cloud.google.com/tpu/docs/intro-to-tpu>`_
+which is Google's customized hardware for accelerating machine learning workloads.
+Two different TPU architectures are available on GCP:
+
+- `TPU Nodes <https://cloud.google.com/tpu/docs/system-architecture-tpu-vm#tpu-node>`_
+- `TPU VMs <https://cloud.google.com/tpu/docs/system-architecture-tpu-vm#tpu-vm>`_
+
+The way to interact with the two architectures can be different.
+For TPU Nodes, a host VM communicates with the TPU host over gRPC.
+For TPU VMs, you can SSH directly into a VM that is physically connected to the TPU device.
+For more details please refer to GCP `documentation <https://cloud.google.com/tpu/docs/system-architecture-tpu-vm#tpu-arch>`_.
+
+
+Getting TPU in one command
```

Comment:
TPU -> a TPU node?

Reply:
I add a new option to launch TPU VM with `sky tpunode --tpu-vm` so now both are supported. I guess the current title is fine now?